src/components/carousel/NavigationButton.tsx

```diff
@@ -8,41 +8,28 @@ export type Direction = 'previous' | 'next';
 interface Props {
   onClick: (direction: Direction) => void;
   direction: Direction;
+  fullScreen: boolean;
 }
 
-const NavigationButton: FC<Props> = ({ direction, onClick }) => {
+const NavigationButton: FC<Props> = ({ direction, onClick, fullScreen }) => {
+  const { button, buttonContainer, icon } = useMultiStyleConfig(
+    'Carousel',
+    fullScreen ? { variant: 'fullScreen' } : {}
+  );
   const side = direction === 'previous' ? { left: '0' } : { right: '0' };
 
   return (
     <chakra.button
       onClick={() => onClick(direction)}
-      position="absolute"
-      top="0"
       {...side}
-      width="lg"
-      height="full"
-      display="flex"
-      justifyContent="center"
-      alignItems="center"
-      color="white"
-      visibility="hidden"
-      _groupHover={{ md: { visibility: 'visible' } }}
       aria-label={`${direction} slide`}
+      __css={buttonContainer}
```

Comment:
I think the navigation buttons should not be visible on mobile. This automatically worked for the small carousel because you cannot hover on mobile. But since they are always visible on the fullscreen variant, we need to hide them on mobile

Reply:
Done